src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public Figure getRandomFigure()
+    public Figure getDefaultFigure()
```

Comment:
The method `getDefaultFigure` is declared but not implemented. You need to provide an implementation that returns a default figure instance.